cdk/visit/lambda_code/log_visit.py

```diff
@@ -84,16 +84,18 @@ def registrationWorkflow(current_user):
         print(e.response['Error']['Message'])
 
 
-def addVisitEntry(current_user):
+def addVisitEntry(current_user, location):
 
     # Get the current date at which the user logs in.
     visit_date = datetime.datetime.now().timestamp()
 
     # Add the item to the table.
     response = visits.put_item(
+        # TODO: Why is the label PK and SK instead of visit_date and username?
```

Comment:
This is a great question! PK and SK stand for partition key and sort key. The combination of the two represent a unique key combination for an associated value. I believe that we actually use a single table to store both users and visits. In the case of a user, the PK == email IIRC, and in the case of visits the PK == an epoch timestamp.

Our table structure is not ideal, and I think ultimately we will want to think about storing data in a more efficiently query-able manner but hope this answers your question for the time being!

Reply:
```suggestion
```

We should remove this `TODO` or replace it with a note explaining the choice like @dggorsen's comment.

Reply:
Just resolved this and the logging note in the latest commit!